addOns/grpc/src/main/java/org/zaproxy/addon/grpc/internal/HttpPanelGrpcView.java

```diff
@@ -145,11 +156,16 @@ public void dataChanged(HttpPanelViewModelEvent e) {
                 httpPanelGrpcArea.setText(protoBufMessageDecoder.getDecodedOuput());
             }
         } catch (Exception er) {
-            httpPanelGrpcArea.setText(er.getMessage());
+            httpPanelGrpcArea.setText(protoBufMessageDecoder.getDecodedOuput() + er.getMessage());
             httpPanelGrpcArea.setBorder(BorderFactory.createLineBorder(Color.RED));
         }
         if (!isEditable()) {
             httpPanelGrpcArea.discardAllEdits();
         }
     }
+
+    public void showInvalidMessageFormatError(String message) {
+        JOptionPane.showMessageDialog(
+                null, message, "Invalid Message Format", JOptionPane.ERROR_MESSAGE);
```

Comment:
- Title should be i18n'ed as well.
- Would be better to set a proper parent, the views are shown in different places (main window, manual request editor).

Reply:
is this work for parent `View.getSingleton().getPopupMenu()`